boards/Pico14/main.py

```diff
@@ -0,0 +1,50 @@
+import board
+
+from kb import KMKKeyboard
+
+from kmk.extensions.lock_status import LockStatus
+from kmk.extensions.stringy_keymaps import StringyKeymaps
+from kmk.keys import KC
+from kmk.modules.layers import Layers
+
+Pico14 = KMKKeyboard()
+
+class LEDLockStatus(LockStatus):
+    def set_lock_leds(self):
+        if self.get_num_lock():
+            Pico14.leds.set_brightness(100, leds=[0])
+        else:
+            Pico14.leds.set_brightness(0, leds=[0])
+
+    def after_hid_send(self, sandbox):
+        super().after_hid_send(sandbox)  # Critically important. Removing this will break lock status.
+
+        if self.report_updated:
+            self.set_lock_leds()
+
+
+Pico14.modules.append(Layers())
+Pico14.extensions.append(LEDLockStatus())
+Pico14.extensions.append(StringyKeymaps())
+
+# Make this for better looking formatting...
+______ = 'NO'
+
+Pico14.keymap = [[
+  # Layer 0 QWERTY
+    'NUMLOCK',  'NUMPAD_SLASH', 'NUMPAD_ASTERISK',
+    'NUMPAD_7', 'NUMPAD_8',     'NUMPAD_9',
+    'NUMPAD_4', 'NUMPAD_5',     'NUMPAD_6',
+    'NUMPAD_1', 'NUMPAD_2',     'NUMPAD_3',
+    'NUMPAD_0', ______,         'NUMPAD_DOT',
+], [
+  # Layer 1
+    'NUMLOCK',  'NUMPAD_SLASH', 'NUMPAD_ASTERISK',
+    'HOME',     'UP',           'PGUP',
+    'LEFT',     'NUMPAD_5',     'RIGHT',
+    'END',      'DOWN',         'PGDN',
+    'INS',       ______,        'DEL',
```

Comment:
Please use the `KC.xxx` pattern for the default configuration. Stringy keymaps are a niche application for special cases and not the recommended for default use.